### DISCUSSION TITLE
希望可以增加一个功能，鼠标选中所对应的文本，通过快捷键一键翻译文本所对应的段落

### DISCUSSION THREAD

**Discussion by chaniqure** [YiiGuxing/TranslationPlugin#5522] [Ideas]
实际上就是Chrome的沉浸式翻译插件的功能，当鼠标光标在某一段文字中，当我按某个快捷键时，这一段文字就直接全部翻译了，不用手动去选择这一段文件，减少了一个操作，我觉得很好用

**Comment by YiiGuxing** [comment]
插件是有类似的功能的，当按下翻译快捷键时会自动拾取光标所在位置周围文字进行翻译的，但它只在一行内进行拾取，整段文字在IDE上没办法实现的，在源代码文件中我们很难判断哪些部分是一个整体的文字段落的，其语义环境远比网页上的内容要复杂得多。
